ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/KeyValueCommittingSpec.scala

```diff
@@ -63,7 +64,11 @@ class KeyValueCommittingSpec extends AnyWordSpec with Matchers {
 
   val dedupKey = DamlStateKey.newBuilder
     .setCommandDedup(
-      DamlCommandDedupKey.newBuilder.addSubmitters(alice).setCommandId(commandId).build
+      DamlCommandDedupKey.newBuilder
+        .addSubmitters(alice)
```

Comment:
In general, you may want to follow the order of fields as declared in the proto file when constructing one so that it's easier to check at a glance which fields you're setting.

Reply:
Done.